muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/chat.scss

```diff
@@ -188,6 +189,16 @@
   width: 400px;
 }
 
+.chat__panel--active {
+  box-shadow: 0 0 15px $color-chat-controlbox-box-shadow;
+  transition: box-shadow 0.3s ease-in-out;
+}
+
+.chat__panel--notactive {

```

Comment:
This is unnecessary as it just repeats the default style defined in `.chat__panel` rule already.